modules/apps/asset/asset-list-web/test/js/components/VariationsNav.test.js

```diff
@@ -252,13 +250,6 @@ describe('VariationsNav', () => {
 		expect(nodes[1].innerHTML).toContain('Anyone');
 
 		expect(await saveVariationsListPriorityService).toHaveBeenCalled();
-
-		expect(global.Liferay.Util.openToast).toHaveBeenCalledWith({
-			message: Liferay.Language.get(
-				'your-request-completed-successfully'
-			),
-			type: 'success',
-		});
```

Comment:
Instead of deleting tests, can't we re-work these to check if `openToast` has been called with those params?

Reply:
How would you mock the exported/imported `openToast`? I haven't figured it out.

Reply:
@bryceosterhaus cc

Reply:
I think you can spy on the module by importing it, [see the docs here](https://jestjs.io/docs/jest-object#jestspyonobject-methodname)

Reply:
I'll play around with it, thanks. I know I tried the regular import and then overriding it, but that didn't work.